### DISCUSSION TITLE
Testing: GitHub Discussions polls

### DISCUSSION THREAD

**Discussion by thibaudcolas** [wagtail/wagtail#8259] [Polls]
This is a [new GitHub Discussions feature](https://github.com/github/feedback/discussions/12821). Let’s see how it works!

**Comment by zerolab** [comment]
Looks like the wheels are spinning on this 😂

**Reply by thibaudcolas** [reply]
Important to not keep things door-mant!

**Reply by lb-** [reply]
How many doors will be in the new version of Wagtail - I must know.

**Reply by lb-** [reply]
Well we know there will be at least one Wheel 🥁

**Comment by NotANormalNerd** [comment]
I think we are asking the wrong questions here and therefor the answers are not leading into any direction. I think we should circle back and review the question in an approved committee that has the skills to figure out the right question. They can take a step back and look at the bigger picture, this will also have implication on the longer strategy of this poll.

Well this and obviously windows.

**Reply by lb-** [reply]
Sounds like a meeting I was in today - were you there?

**Reply by NotANormalNerd** [reply]
Are you also working for megacorp inc.?

**Comment by stevedya** [comment]
Where can I pickup my "I voted" sticker? 
Or does it just come in the mail?

Thanks in advance.

**Reply by thibaudcolas** [reply]
Hi @stevedya, thank you for having voted. Here is your sticker, please make the most of it.

![iv-ted](https://user-images.githubusercontent.com/877585/162067250-8c077496-f18b-429f-ab57-3cc0e3948217.png)

**Reply by stevedya** [reply]
@thibaudcolas 
### Thanks! I printed it out today and stuck it on my hand, heres a picture I took in my office a few minutes ago. 
<img src="https://user-images.githubusercontent.com/25041665/163490073-200e2797-1304-481a-b354-fd61d0224df7.jpg" width="600px"/>
